src/app/purpose/page.tsx

```diff
@@ -12,60 +12,64 @@ export default function page() {
       </div>
       {/* engaging creative kids block */}
       <div className="bg-secondary-blue pb-10">
-        <div className="bg-neutral-white rounded-2xl p-10 md:p-5 shadow-md flex flex-col md:flex-row items-start mx-auto w-11/12">
+        <div className="bg-neutral-white rounded-2xl px-10 pt-10 pb-10 lg:pb-[60px] md:p-5 shadow-md flex flex-col md:flex-row items-start mx-auto w-11/12">
```

Comment:
Could use `py-10` here

Reply:
Consider using `lg:pb-16` instead of `lg:pb-[60px]`. I'm aware that this will not render exactly 60px but its better to use the underlying `rem` units that tailwind's utility classes are based on.